tket/README.md

```diff
@@ -10,7 +10,7 @@ Assuming the prerequisite tools and configuration described in the top-level
 README, to build `tket` (from the root directory of the repo):
 
 ```shell
-conan create tket --user=tket --channel=stable --build=missing -o "boost/*":header_only=True
+conan create tket --user=tket --channel=stable --build=missing -o boost/*:header_only=True
```

Comment:
I recall that there was a reason for these quotes around `boost/*`, that they were necessary on MacOS systems running zsh.